web/views.py

```
@@ -4876,19 +4900,48 @@ def send_donation_thank_you_email(donation):
+                logger.error(f"Error verifying payment intent: {str(e)}")
+                # Continue to show the success page even if verification fails
+                # The webhook will eventually update the status
+
+        if donation.status == "completed":
             context = {
                 "donation": donation,
             }
             return render(request, "donation_success.html", context)
-        except Donation.DoesNotExist:
-            pass
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)